### ISSUE TITLE
using with kinect v2 (iai_kinect)

### ISSUE THREAD

**Issue by rybandrei2014** [felixendres/rgbdslam_v2#78]
Hello, i noticed that most launch files are using openni, can i use them with kinect2_bridge?

**Comment by zouyajing** [comment]
Yes. Just change the parameters of topic in launch file. I have tried and it works.

**Comment by wiibrew** [comment]
I tried to configure the launch files but fails. @zouyajing  can you share a launch file for kinect 2 running rgbdslam.
